docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -261,7 +261,7 @@ Native executables make Quarkus applications ideal for containers and serverless
 
 Make sure to have `GRAALVM_HOME` configured and pointing to GraalVM version {graalvm-version}.
 
-Create a native executable using: `./gradlew buildNative`.
+Create a native executable using: `./gradlew quarkusBuild -Dquarkus.package.type=native`.
```

Comment:
No biggie, but I wonder if we shouldn't promote using `./gradlew build` instead. Seems easier to remember and well-understood by Gradle users I think

Reply:
yes, you right, I'm updating it.

Reply:
Should we also make `buildNative` depends on `build` instead of `quarkusBuild` ?

Reply:
Hm, I think in this case depending on `quarkusBuild` may be better, but maybe it doesn't matter, WDYT?

Reply:
Well, yes, as we aim to remove `buildNative` task, this may be better to keep it as an "alias" of `quarkusBuild -Dquarkus.package.type=native` and not make it depends on something else which actually depends on `quarkusBuild`